xknxproject/xml/parser.py

```diff
@@ -162,6 +165,19 @@ def parse(self, language: str | None = None) -> KNXProject:
             locations=space_dict,
         )
 
+    def convert_functions(self, functions: XMLFunctions) -> list[str, Functions]:
```

Comment:
the return type annotation is off.
```suggestion
    def convert_functions(self, functions: XMLFunctions) -> list[Functions]:
```